docs/001_develop/03_client-capabilities/008_interaction/number-field.mdx

```diff
@@ -0,0 +1,357 @@
+<td>
+
+```typescript
+<rapid-number-field maxlength="${(x) => x.maxLength}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>minlength</td>
+      <td><code>number</code></td>
+      <td>The minimum number of characters allowed</td>
+<td>
+
+```typescript
+<rapid-number-field minlength="${(x) => x.minLength}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>max</td>
+      <td><code>number</code></td>
+      <td>Defines the maximum value allowed</td>
+<td>
+
+```typescript
+<rapid-number-field max="${(x) => x.max}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>min</td>
+      <td><code>number</code></td>
+      <td>Defines the minimum value allowed</td>
+<td>
+
+```typescript
+<rapid-number-field min="${(x) => x.min}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>withFormatting</td>
+      <td><code>boolean</code></td>
+      <td>Enables you to format the number, use <code>maximumFractionDigits</code> to set the number of decimal places.</td>
+<td>
+
+```typescript
+<rapid-number-field withFormatting :options=${() => ({maximumFractionDigits: 2})}>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>name</td>
+      <td><code>string</code></td>
+      <td>Defines the name of the element</td>
+<td>
+
+```typescript
+<rapid-number-field name="username">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>placeholder</td>
+      <td><code>string</code></td>
+      <td>Sets placeholder text for the element (which disappears when the user starts typing)</td>
+<td>
+
+```typescript
+<rapid-number-field placeholder="Enter your name">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>readonly</td>
+      <td><code>boolean</code></td>
+      <td>If true, the user cannot change the value of this field</td>
+<td>
+
+```typescript
+<rapid-number-field ?readonly="${(x) => x.isReadonly}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>size</td>
+      <td><code>number</code></td>
+      <td>Sets the width of the component</td>
+<td>
+
+```typescript
+<rapid-number-field size="${(x) => x.fieldSize}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>step</td>
+      <td><code>number</code></td>
+      <td>Defines the amount the field value changes each time the user clicks to increase or decrease the field value</td>
+<td>
+
+```typescript
+<rapid-number-field type="number" step="${(x) => x.stepValue}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>value</td>
+      <td><code>string</code></td>
+      <td>Sets the value for this component. Use the `sync` utility to bind the data the template back to the model.</td>
+<td>
+
+```typescript
+<rapid-number-field value="${sync((x) => x.fieldValue)}">
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+### Properties
+
+This component doesn't have any properties which are not also controlled via attributes.
+
+### Slots
+
+| Slot | Use |
+|---|---|
+| start | Start slot. |
+| end | End slot. |
+
+### Parts
+
+| Part |
+|---|
+| label |
+| start |
+| controls |
+| end |
+
+### Events Fired
+This component doesn't emit any events.
```

Comment:
Does this not emit change events? (I haven't checked, but intuitively I thought it would have)